src/common/interfaces.ts

```diff
@@ -2,3 +2,64 @@ export interface IConfig {
   get: <T>(setting: string) => T;
   has: (setting: string) => boolean;
 }
+
+export enum OperationStatus {
+  PENDING = 'Pending',
+  IN_PROGRESS = 'In-Progress',
+  COMPLETED = 'Completed',
+  FAILED = 'Failed',
```

Comment:
shouldn't we add the expired status to the enum?

Reply:
added (though its irrelevant as it isn't used).
**this was copied from the job-manager code, it doesn't have it in the enum either**